Makefile

```diff
@@ -109,6 +109,9 @@ test-docker-py: build ## run the docker-py tests
 test-integration-cli: build ## run the integration tests
 	$(DOCKER_RUN_DOCKER) hack/make.sh dynbinary test-integration-cli
 
+test-splunk-logger: build ## run the logger tests
```

Comment:
Yeah, I agree -- it's a little strange to have the log driver in contrib, but still have bits in `Makefile` and `hack/make` supporting it.  Is there a way we could instead have a script in `contrib` instead which bootstraps the driver into the codebase?